### ISSUE TITLE
Translation: Slovenian translation

### ISSUE THREAD

**Issue by JadranR** [Timthreetwelve/WUView#69]
### Report a problem with a translation

[Strings.sl-SL.zip](https://github.com/Timthreetwelve/WUView/files/13693602/Strings.sl-SL.zip)
Slovenian translation for Windows Update Viewer 0.5.26.0 (inside ZIP Archive)

### Contribute a translation

```shell
This attached file is My Slovenian translation for Windows Update Viewer 0.5.26.0, per Today.

I wish you Merry Chrismass and Happy New Year 2024, too :-)

Jadran Rudec
```

### If this is a contribution, enter the language name and code

si-SL

### If you wish to have your contribution acknowledged

Jadran Rudec

**Comment by Timthreetwelve** [comment]
@JadranR 

What a wonderful surprise! Thank you very much! ❤️

Your translation will be in the next release, which will be sometime after the new year. 🎉

Happy holidays! 

Tim

**Comment by Timthreetwelve** [comment]
I'm sorry that it took me so long to get this released. 

Implemented in version 0.6.0.
